cms/djangoapps/contentstore/views/item.py

```diff
@@ -1036,7 +1038,12 @@ def _get_release_date(xblock, user=None):
     Returns the release date for the xblock, or None if the release date has never been set.
     """
     # If year of start date is less than 1900 then reset the start date to DEFAULT_START_DATE
-    if xblock.start.year < 1900 and user:
+    try:
+        xblock_start_year = xblock.start.year
```

Comment:
Can you leave a comment in the code to that effect? Otherwise it seems confusing why that would raise a `ValueError`